MovieApp/Views/HomeScreen/DefaultHomeScreen/HomeScreenTableViewCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  HomeScreenTableViewCell.swift
+//  MovieApp
+//
+//  Created by Nikola Đokić on 15.04.2022..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+import MovieAppData
+
+class HomeScreenTableViewCell: UITableViewCell{
+    
+    static let id = "homeScreenViewCell"
```

Comment:
You can use: 
`static let identifier = String(describing: HomeScreenTableViewCell.self)`
to define you reuse identifiers.